core/arch/arm/include/sm/sm.h

```diff
@@ -120,4 +120,18 @@ void *sm_get_sp(void);
  */
 void sm_init(vaddr_t stack_pointer);
 
+#ifndef CFG_SM_PLATFORM_HANDLER
+/*
+ * Returns false if we handled the monitor service and should now return
+ * back to the non-secure state
+ */
+static inline bool sm_platform_handler(struct sm_ctx *ctx)
+{
+	(void)ctx;
```

Comment:
Please use `__unused` (from `<compiler.h>`) where the parameter is declared instead.